doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,43 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the Networking core image
+******************************************************
+
+Apart from capability of the recovery of the application core image, also capability of recovery of the networking core
+is avielable. The feature is possible to be used if nRF53 MCUboot was build either with:
+
+* the "Simultaneous multi-image DFU" option enabled (:kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE`)
+* the "Serial recovery for the NET-core in the one image pair mode" opton enabled (:kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE`).
+
+In order to upload the networking image need to specify it's slot explictly by 3, for instance using mcumgr cli image upload -n option::
+
+     ./mcumgr image upload <build_dir_path>/zephyr/net_core_app_update.bin -e -n 3 -c serial_conn
+
+where serial_conn is the serial connection configuration.
+
+To enable serial recovey of the net core while multi-image update is not enabled in the MCUboot, set the following option:
+
+* select :kconfig:option:`BOOT_IMAGE_ACCESS_HOOK`
+* select :kconfig:option:`FLASH_SIMULATOR`
+* select :kconfig:option:`FLASH_SIMULATOR_DOUBLE_WRITES`
+* disable :kconfig:option:`FLASH_SIMULATOR_STATS`
+* select :kconfig:option:`MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD`
+* and fillanly select :kconfig:option:`NRF53_RECOVERY_NETWORK_CORE`
+
+Additionally, the memory partitions must be defined and include:
+
+* ``mcuboot_primary`` and ``mcuboot_secondary`` partitions for the application core image slots.
+* ``mcuboot_primary_1`` partitions for the network core image slot.
+* ``pcd_sram`` partition used for command exchange between the application core and the network core (see :kconfig:option:`CONFIG_PCD_APP`).
+
+.. note::
+
+   When MCUboot is used with :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` enabled
+   The application core does not have direct access to the network core flash memory.
+   This is similar as for Simultaneous multi-image DFU, so is using
+   the ``mcuboot_primary_1`` RAM partition mediator.
```

Comment:
```suggestion
   When using MCUboot with the :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` option enabled,
   the application core does not have direct access to the network core flash memory.
   Use ``mcuboot_primary_1`` as the RAM partition mediator.
```

Reply:
This is explanation of reason because of which mcuboot_primary_1 must exist.
The suggestion changes the meaning.

Reply:
Sorry for messing up the meaning. How about then something like: "Due to this, ``mcuboot_primary_1`` is used as the RAM partition mediator".

Reply:
applied